### PR TITLE
Fixes First Ctrl+R and Ctrl+F not showing long name variables correctly

### PR DESCRIPTION
**Pull request by piyushdagar**
Fixes First Ctrl+R and Ctrl+F not showing long name variables correctly #26982

### CODE REVIEW

File: scene/gui/line_edit.cpp [hunk 1]
```diff
@@ -1180,7 +1180,8 @@ void LineEdit::set_cursor_position(int p_pos) {
 				}
 				if (accum_width > window_width)
 					break;
-
+				if(accum_width < window_width)
```
Comment (akien-mga) [inline scene/gui/line_edit.cpp]: Style issue: https://travis-ci.org/godotengine/godot/jobs/505833950

**Comment by piyushdagar** [comment]
@akien-mga

**Comment by piyushdagar** [comment]
@akien-mga

**Comment by akien-mga** [comment]
Could you amend the commit log to state what it does in the first line (commit title)? That would be more explicit than just the issue number when reading the `git log`.

Just inverting the two lines should do it:
```
Fix LineEdit not showing long strings correctly

Fixes #26982
```
Or anything that better describes the actual fix, and not just the original issue (Ctrl+F or R in the search bar are not really related to the actual fix. The commit should describe what the fix does, not one of the symptoms of the bug it fixed).

**Pull request closed without merging**